OWML.ModLoader/ModDep.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OWML.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.ModLoader
+{
+    public class Mod
```

Comment:
Like (I think) @Raicuparta suggested, you can change both the filename and class name to ModDependency

Reply:
i... did? probably didn't save when I switched branches...

Reply:
oh wait, this was before the commit